edx-olive/configuration-old
# SE-2585 Add systemd override to auto restart mongod

This ensures mongod is always restarted if it crashes or exits for some reason.

**Test instructions**:

- deploy this to a stage mongo node using ansible:
  - ssh to the stage bastion
  - `cd ~/configuration`
  - check branch and pull latest changes
  - run the `mongo.yml` ansible playbook against a stage mongo node
- ssh into the mongo node 
- `systemctl status mongod` and verify that running correctly
- `sudo pkill mongod`
- immediately (within 5 seconds) run `systemctl status mongod` again and verify that it's exited
- wait 10 seconds and run `systemctl status mongod` again and verify that mongod is running again 

**Reviewers**:
